src-v2/Not/Not.Blazor/TM/NotConsole.razor

```diff
@@ -0,0 +1,40 @@
+﻿<MudDialog TitleClass="mud-console-header py-0 " Class="mud-console" ContentClass="pa-1">
+    <DialogContent>
+        <MudText Typo="Typo.overline" Class="mud-console-output">
+            @foreach (var line in OutputLines)
+            {
+                <div>@line</div>
+            }
+        </MudText>
+    </DialogContent>
+</MudDialog>
+
+@code {
+    [Parameter]
+    public List<string> OutputLines { get; set; } = new List<string>();
+
+    public void AddOutput(string newLine)
+    {
+        OutputLines.Add(newLine);
+        InvokeAsync(StateHasChanged);
+    }
+
+    public void ClearConsole()
```

Comment:
Is there a reason for those methods not to be async? While it is technically correct, as the async code runs last and it would behave as expected, it would be better to use async Task and await, because otherwise you might one day modify the code and if you omit async it may start to behave weirdly.